package/MDAnalysis/analysis/gnm.py

```diff
@@ -169,39 +154,61 @@ def order_list(w):
 
 
 class GNMAnalysis(object):
-    '''Basic tool for GNM analysis.
+    """Basic tool for GNM analysis.
 
     Each frame is treated as a novel structure and the GNM
     calculated.  By default, this stores the dominant eigenvector
     and its associated eigenvalue; either can be used to monitor
     conformational change in a simulation.
-    '''
 
-    def __init__(self, universe, selection='protein and name CA', cutoff=7.0, ReportVector=None, Bonus_groups=()):
+    Parameters
+    ----------
+    universe : Universe
+          Analyze the full trajectory in the universe.
+    selection : str (optional)
+          MDAnalysis selection string, default "protein and name CA"
+    cutoff : float (optional)
+          Consider selected atoms within the cutoff as neighbors for the
+          Gaussian network model.
+    ReportVector : str (optional)
+          filename to write eigenvectors to, by default no output is written
+          (``None``)
+    Bonus_groups : tuple
+          This is a tuple of selection groups, the com of each which will be
```

Comment:
You should specify  that this has to be selection strrings. Giving an atomgroup doesn't work and will likely produce a Attribute error